CYF-LONDON | Giosef Ferraro | HTML-CSS-Module | Week1

Giosef92 opened the pull request:
CYF-LONDON | Giosef Ferraro | HTML-CSS-Module | Week1

Complete the task list below this message.
If your PR is rejected, check the task list.


## Learners, PR Template

Self checklist

- [ x] I have committed my files one by one, on purpose, and for a reason
- [x ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ x] I have tested my changes
- [ x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [x ] My changes meet the [requirements](./README.md) of this task

## Changelist

SallyMcGrath reviewed (changes requested):
Hi @Giosef92 thanks for this!

1. Please title your PR as everyone else has - take a look on the PR list on this repo.
2. You have written a LOT of CSS here. As I read through it, it seems like you could delete quite a lot of it. This programme isn't focused on CSS, or front end, but it is focused on testing our code and writing high quality code. 

With this in mind, please go through your CSS -- you can use this branch preview if you like https://deploy-preview-185--cyf-onboarding-module.netlify.app/wireframe/

And find me 20 lines you could delete. And then delete them! :) For extra value, tell me why you could delete them and how you figured this out. 

Hint, I would start with some of these width rules. Good luck!

Giosef92 commented:
Refactor CSS: Cleanup, Fix Layout, and Improve Consistency
- Removed redundant styles and fixed inconsistent indentation.
- Optimized header padding declaration.
- Fixed layout issues in .articles and .sub-article links.
- Improved responsiveness and removed unnecessary media queries.
- Ensured consistent styling in footer and images. @SallyMcGrath

alastair87 commented:
### Semantic HTML tags are used to structure the webpage.

You have correcctly used semantic HTML to structure the web page including three articles with the structure requested.

###  The page scores 100 for Accessibility in the Lighthouse audit.

You haven't met this requirement. You have a score of 93. Please review the Lighthouse accessibility score and see what you can do to fix it (see link above). Accessibility is a core requirement to make sure your web site is available to everyone who wants to use it. You're on the right track but a couple of easily fixable issues are highlighted.

### The page header includes a title and description.

Requirement met.

###  The articles section has three unique articles, each including a title, summary, and a link.

You've done this. Your CSS is a bit odd because you've styled the left image with the "second-img"  class but there is no class applied to the third one.

 ### The page footer is fixed to the bottom of the viewport.

Done.

 ### The webpage is styled using a linked .css file.

Done.

 ### The webpage is properly committed and pushed to a branch on GitHub.

Done.
